common-util/Login/config.jsx

```diff
@@ -84,3 +85,29 @@ export const SUPPORTED_CHAINS_MORE_INFO = SUPPORTED_CHAINS.map((chain) => {
 
   return { id, networkDisplayName: name, networkName: getNetworkName() };
 });
+
+/**
+ * Solana supported chains
+ */
+export const SOLANA_SUPPORTED_CHAINS = [
+  {
+    id: null,
+    networkDisplayName: 'Solana',
+    networkName: 'solana',
+    blockchainName: 'solana',
+  },
+  {
+    id: null,
+    networkDisplayName: 'Solana Devnet',
+    networkName: 'solana-devnet',
+    blockchainName: 'solana',
+  },
+];
+
+/**
+ * Returns the list of all supported chains.
+ */
+export const ETHEREUM_AND_SOLANA_SUPPORTED_CHAINS = [
```

Comment:
Should really be `SUPPORTED_CHAINS`

Reply:
If that's too hard to update rn because of the conflicting name, maybe just `ALL_SUPPORTED_CHAINS`.